README.md

```diff
@@ -22,15 +22,64 @@ With [npm](http://npmjs.org) do:
 ## Filters
 
 <a id="tokenize"></a>
-### tokenize: *word* | *treebank* | */regex_expression/*
-tokenise a string
+### anglicize: *true*
+Anglicize a string
 
 ```javascript
 
 	var stylesheet = {
-           "set": "my dog hasn't any fleas.",
-           "tokenize" : "Treebank"
-    };
-     JBJ.render(stylesheet, console.log);
+    "set": "ÂÇİĞÖŞÜÑ âçığöşüñ",
+    "anglicize": true
+  };
+  JBJ.render(stylesheet, console.log);
+  // ACIGOSUN acigosun
+
+```
+
+### countCharacters: *true* | *slug* | */regex_expression/*
```

Comment:
Please add `<a id="countCharacters"></a>`.
And do that before all actions.
This is the link used by JBJ Playground.
And if you don't add it, the automatic anchor will include the whole section title, including parameters.